guides/source/form_helpers.md

```diff
@@ -726,7 +726,7 @@ The following two forms both upload a file.
 <% end %>
 ```
 
-Rails provides the usual pair of helpers: the barebones `file_field_tag` and the model oriented `file_field`. As you would expect in the first case the uploaded file is in `params[:picture]` and in the second case in `params[:person][:picture]`.
+Rails provides the usual pair of helpers: the barebones `file_field_tag` and the model oriented `file_field`. As you would expect in the first case the uploaded file is in `params[:person][:picture]` and in the second case in `params[:picture]`.
```

Comment:
This was correct before: "in the first case" refers to the first helper mentioned in the previous sentence, not the first example in the code block above.

Reply:
D'oh!  You're right.  I'm working on cleaning up this section right now anyway though.  There are several examples that don't match up with their accompanying paragraphs.  I'll fix this with it.

Reply:
thanks for catching that @eugeneius.  Maybe just flip the examples in the code block, so that everything (example code block and references to `field_field_tab and `field_field` are all consistent.